tyk-headless/configs/tyk_mgmt.conf

```diff
@@ -1,6 +1,5 @@
 {
     "listen_port": 8080,
-    "control_api_port": 9696,
```

Comment:
Is this compatible with the default values (i.e. if someone upgrades to this, would their control port change)?

Reply:
Yes it would.  But I think it is a necessary breaking change because it contradicts what's in the values.yaml

if tyk-headless/values.yaml `control-port.enabled` is set to `false`, it will still get set because of the hard core above